contrib/kind.sh

```diff
@@ -391,7 +391,7 @@ if [ "${GITHUB_ACTIONS:-false}" == "true" ]; then
       -e 's/^.*kubernetes cluster\.local/& net/' \
       -e '/^.*upstream$/d' \
       -e '/^.*fallthrough.*$/d' \
-      -e '/^.*forward . \/etc\/resolv.conf$/d' \
+      -e '/^.*forward . 192.168.1.1' \
```

Comment:
this only exists in a home network, you need to forward to a public DNS like google 8.8.8.8

Reply:
changed to 8.8.8.8